binder/src/main/java/io/grpc/binder/SecurityPolicies.java

```diff
@@ -97,6 +101,55 @@ public static SecurityPolicy hasSignature(
         packageManager, packageName, ImmutableList.of(requiredSignature));
   }
 
+  /**
+   * Creates a {@link SecurityPolicy} which checks if the given package is signed with the same
+   * certificate that the current Android platform was signed with.
+   *
+   * @param packageName the package name of the allowed package.
+   * @param backgroundExecutor an executor suitable for blocking disk I/O.
+   * @throws NullPointerException if any of the inputs are {@code null}.
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/11238")
+  public static AsyncSecurityPolicy hasSameSignatureAsPlatform(
+      PackageManager packageManager, String packageName,
+      ListeningExecutorService backgroundExecutor) {
```

Comment:
It's usually preferred to take a regular `Executor` here and then use Futures.submit() internally. 

Also very nit, but while we're here the google formatter would put each of these parameters on its own line if it doesn't fit into a single line, rather than placing PackageManager and String on the same line.

Reply:
Updated. Also ran `google-java-format`, except for the unrelated lines.